### PR TITLE
Create view profile page

### PR DESCRIPTION
**Pull request by Tyak99**
### What does this PR do?

* Displays users profile on the profile page

### Description of Task to be completed?

* update eslintrc file to have jest config
* create header component
* create sidebar component
* create component to hold user details
* write test for components
* write action testing
* add profile route `/profile`
* add fileMock file to mock stylesheets for jest tests

### Any background context you want to provide?

N/A

### What are the relevant Github issues?

N/A

### Questions:

N/A

### CODE REVIEW

File: src/components/Sidebar/Sidebar.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+import './Sidebar.scss';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import PropTypes from 'prop-types';
+
+const Sidebar = ({ children }) => (
+  <aside className="aside">
+    <div className="profileImage">
+      <img
+        src="https://cdn.umnbootcamp.com/wp-content/uploads/sites/93/2018/03/placeholder-person.png"
+        alt=""
```
Comment (fortunee) [inline src/components/Sidebar/Sidebar.js]: alt shouldn't be empty

File: src/components/Sidebar/Sidebar.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import './Sidebar.scss';
```
Comment (fortunee) [inline src/components/Sidebar/Sidebar.js]: use lowercase here `'./sidebar.scss'`

**Review by fortunee — COMMENTED** [review]
Great, please review my comments

**Review by beejay1293 — COMMENTED** [review]
Looks good Babatunde. nice work.

**Review by fortunee — APPROVED** [review]
👍

**Pull request merged**